src/algokit/cli/init.py

```diff
@@ -429,6 +439,52 @@ def _maybe_git_init(project_path: Path, *, use_git: bool | None, commit_message:
         _git_init(project_path, commit_message=commit_message)
 
 
+def _maybe_move_github_folder(*, project_path: Path, use_workspace: bool) -> None:
+    """Move contents of .github folder from project_path to the root of the workspace if exists
+    and the workspace is used.
+
+    Args:
+        project_path: The path to the project directory.
+        use_workspace: A flag to indicate if the project is initialized with workspace flag
+    """
+
+    source_dir = project_path / ".github"
+
+    if (
+        not use_workspace
+        or not source_dir.exists()
+        or not (workspace_root := get_workspace_project_path(project_path.parent))
+    ):
+        return
+
+    target_dir = workspace_root / ".github"
+
+    for source_file in source_dir.rglob("*"):
+        if source_file.is_file():
+            target_file = target_dir / source_file.relative_to(source_dir)
+
+            if target_file.exists():
+                logger.debug(f"Skipping move of {source_file.name} to {target_file} (duplicate exists)")
+                continue
+
+            try:
+                target_file.parent.mkdir(parents=True, exist_ok=True)
+                shutil.move(str(source_file), str(target_file))
+            except shutil.Error as e:
+                logger.debug(f"Skipping move of {source_file} to {target_file}: {e}")
+
+    # if source dir does not contain any files or only contains empty folders
+    # then remove it
+    if all(not p.is_file() for p in source_dir.rglob("*")):
+        shutil.rmtree(source_dir)
+        logger.debug(f"No files found in .github folder after merge. Removing `.github` directory at {source_dir}...")
+        return
+
+    click.echo(
```

Comment:
It appears that this will only render if there are files found in the project .github folder.
To me this should be printed always at the start and we should replace this message with something like below:

`Failed to move all files within your project's `.github` folder to the workspace root. Please review any files that remain in your project's `.github` folder and manually include in the root `.github` directory as required.`